wikipedia-dark.user.css

```diff
@@ -384,11 +413,297 @@ regexp("https?:\/\/.*wik(i|t).*(org|jp).*") {
+    	margin: 4px 0 8px;
+   	 	padding: 4px 8px 4px 26px;
+		filter: invert(1) hue-rotate(180deg);
+	}
+	#p-lang .uls-settings-trigger {
+		filter: invert(1) hue-rotate(180deg);
+		background-color: transparent !important;
+		border-color: transparent !important;
+	}
+	.mw-parser-output .dip-tab.selected {
+		background-color: var(--blue-3);
+	}
+	.mw-parser-output .dip {
+		background-color: transparent;
+	}
+	.mw-parser-output .tpl-badge {
+		background-color: var(--blue-3);
+	}
+	.mw-parser-output .wmf-engineering-project-box {
+		background-color: var(--gray-2);
+	}
+	.mw-parser-output .infobox td {
+		border-bottom-color: var(--gray-69) !important;
+	}
+	.mw-parser-output .hatnote {
+		color: var(--gray-a);
+		border-color: var(--gray-69);
+	}
+	table.plainlinks[style="text-align:center; background-color: #f8f9fa;"] {
+		background-color: var(--blue-2) !important;
+	}
+	.site-header {
+		background-color: transparent;
+		border-bottom-color: transparent;
+		position: relative;
+	}
+	.site-header::before {
+		content: '';
+		position: absolute;
+		width: 100%;
+		height: 100%;
+		background-color: var(--gray-2);
+		opacity: .38;
+		backdrop-filter: blur(1rem);
+		-webkit-backdrop-filter: blur(1rem);
+		border-bottom: 1px solid var(--gray-3);
+	}
+	.site-header * {
+		z-index: 1;
+	}
+	.site-header .main-navigation .menu {
+		background-color: transparent !important;
+	}
+	article.post .entry-footer, #cta {
+		background-color: var(--blue-3);
+		color: var(--white);
+	}
+	#cta2 .widget {
+		background-color: var(--blue-3);
+	}
+	.wp-block-rss__item-author, .wp-block-rss__item-publish-date, #wpdcom .wpd-thread-head .wpd-thread-info {
+		color: var(--gray-a);
+	}
+	.photo-credit-container .text-container, article.grid-post .entry-meta, article.grid-post .entry-meta a, #wpdcom .wpd-sbs-toggle, #wpdcom .wpd-sbs-toggle i.fa-caret-down {
+		color: var(--gray-9a);
+	}
+	ul#menu-footer-links, input#mw-searchButton, input[type="hidden"] {
+		background-color: transparent !important;
+	}
+	.photo-credit-container .img-container {
+		border-color: var(--blue-3);
+	}
+	#comments {
+		color: var(--gray-c);
+	}
+	#wpdcom .wpd-form-head, #wpdcom .wpd-thread-head {
+		border-color: var(--gray-3);
+	}
+	#wpdcom .wpd-thread-head .wpd-thread-info {
+		border-color: var(--gray-d);
+	}
+	.wpdiscuz_select {
+		background-color: var(--gray-1) !important;
+		border-color: var(--gray-69) !important;
+	}
+	.mw-highlight .nv {
+		color: var(--blue-1);
+	}
+	[style="background-color: #DFD;border: 1px solid #6B6; font-size: 85%;padding: 5px 8px;border-radius:10px;"] {
+		border-color: var(--green) !important;
+		background-color: var(--green-2) !important;
+	}
+	table.mw-collapsible.mw-collapsed.mw-made-collapsible[style="width:100%;margin:0;padding:0;border:1px solid #999999;border-collapse:collapse;background:#FFFFFF"], table.mw-collapsible.mw-made-collapsible[style="width:100%;margin:0;padding:0;border:1px solid #999999;border-collapse:collapse;background:#FFFFFF"] {
```

Comment:
again here and everywhere, this is overill to match the whole inline style.

In all such areas ONLY use the bits you need, and this WAY too heavy selectors is 100% going to break something else.

cannot accept this at all